torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -268,112 +269,16 @@ void runPreAutodiffPassPipeline(std::shared_ptr<Graph>& graph) {
 
     EliminateCommonSubexpression(graph);
     GRAPH_DEBUG(
-        "After EliminateCommonSubexpression, before CheckInplace\n", *graph);
-
+        "After EliminateCommonSubexpression, before HoistCommonExpression\n",
+        *graph);
+    HoistCommonExpression(graph);
+    GRAPH_DEBUG("After HoistCommonExpression, before CheckInplace\n", *graph);
     CheckInplace(graph);
   }
   GRAPH_DEBUG(
       "After CheckInplace (end of runPreAutodiffPassPipeline)\n", *graph);
 }
 
-void runDiffGraphPasses(std::shared_ptr<Graph>& graph) {
```

Comment:
Unused function

Reply:
wtf. Errr... I'll check the history tomorrow